src/hooks/useHttpClient.tsx

```diff
@@ -6,8 +6,8 @@ import axios, {
 } from 'axios';
 import { useAuth } from './useAuth';
 import { APIClient } from '@lifeomic/one-query';
-import { RestAPIEndpoints } from '../types/rest-types';
```

Comment:
Can we now delete the types defined in this file?

Reply:
Yep, +1 -- there are several big type defs defined in this project that should be deleted.
- fhir api types
- rest api types
- auth api types

Reply:
I had a bunch of test errors come up when deleting and moving over so I assumed we needed both but will remove.